docs/release_process.md

```diff
@@ -34,13 +34,8 @@ See https://semver.org/ .
 
 You must do 3 things to formally release a version:
 
-1. Update `CHANGELOG.md`. Use the following git command to generate the appropriate
-   data format:
-   ```
-   git log --oneline ${PREVIOUS_VER}..upstream/master --no-merges --format=" - %s (%h) by %aN &lt;%aE&gt;"
-   ```
-   where `${PREVIOUS_VER}` is the last release tag, eg `v0.9.3`. Please see previous
-   releases for the final formatting.
+1. Mark the release in the CHANGELOG by replacing the `## Unreleased` header
+   with `## [VERSION] date`.
```

Comment:
```suggestion
   with `## [VERSION] [DATE]`.
```
Maybe this would be more clear?

Reply:
Maybe like this? `[version] data` is the format the changelog has.

```suggestion
   with `## [VERSION] date`, e.g. `## [0.10.0] 2020-04-21`.
```

Reply:
Ah yeah sorry I got confused. The example helps.